packages/omgx/wallet/contracts/LP/interfaces/iL2LiquidityPool.sol

```diff
@@ -11,30 +11,44 @@ interface iL2LiquidityPool {
      *       Event      *
      ********************/
     
-    event ownerAddERC20Liquidity_EVENT(
+    event addLiquidity_EVENT(
```

Comment:
this event naming is I think unconventional. we already know it's an event because it's the structure type. can we drop _EVENT?

Reply:
let's update these events in accordance to the events we have in the contracts